Dockerfile

```diff
@@ -6,11 +6,8 @@ ENV KAFKA_VERSION=2.3.0 SCALA_VERSION=2.12
 RUN set -ex; \
   export DEBIAN_FRONTEND=noninteractive; \
   runDeps='netcat-openbsd'; \
-  buildDeps='curl ca-certificates gnupg dirmngr apt-transport-https'; \
+  buildDeps='curl ca-certificates gnupg dirmngr'; \
   apt-get update && apt-get install -y $runDeps $buildDeps --no-install-recommends; \
-  echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list; \
```

Comment:
This is not even needed for SaaS, right?

Reply:
This is need for SaaS. will create a new image for the same. The image size is 726MB and is taking time to fetch it from docker hub. After removin this package, image size is reduced to 278MB.

Reply:
Ok. Interesting that it is needed for SaaS.